finality-aleph/src/testing/justification.rs

```diff
@@ -167,18 +166,20 @@ fn prepare_env(
     let info_provider = SessionInfoProviderImpl::new(SESSION_PERIOD, verification_policy);
     let finalizer = MockedBlockFinalizer::new();
     let requester = MockedBlockRequester::new();
-    let config = JustificationHandlerConfig::new(request_policy);
-    let jrd = config.justification_request_delay.clone();
+    let config = JustificationHandlerConfig::test();
+    let jrs = JustificationRequestSchedulerImpl::new(request_policy);
```

Comment:
nit, but not a fan of these abbreviated names, they are hard to read / follow - imho code is for humans, bytes are cheap - just spell it out verbose `justification_request_scheduler` etc (it all ends up as bytecode in the end)

Reply:
Done